VideoLocker/src/main/java/org/edx/mobile/util/Config.java

```diff
@@ -402,6 +412,17 @@ public EnrollmentConfig getCourseDiscoveryConfig() {
         }
     }
 
+    public WebViewConfig getWebViewConfig() {
```

Comment:
The `WEBVIEW` variable would also be unused after removing this method. It looks like constants are only defined for object keys, and not for field names. This should probably be standardized one way or another at some point.